UpcomingMovies/UpcomingMovies/UpcomingMovies/Models/Movie.cs

```diff
@@ -0,0 +1,28 @@
+﻿using MvvmHelpers;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace UpcomingMovies.Models
+{
+     public class Movie: ObservableObject
+     {
+        public string Title { get; set; }
+
+        [JsonProperty(PropertyName = "genre_ids")]
```

Comment:
You can use a `ContractResolver`, so you don't need to put serialization code in your models